react/src/Admin/components/UserRow.tsx

```diff
@@ -125,14 +98,15 @@ export default function UserRow(props: {
                 </ListItem>
                 <Collapse in={open}>
                     <Divider />
-                    <UserDetails
-                        user={user}
-                        groups={groups}
-                        onSave={props.onSave}
-                        onDelete={props.onDelete}
-                        loading={loading}
-                        onMinioReset={handleMinioReset}
-                    />
+                    {open && (
+                        <div id={props.user.username}>
+                            <UserDetails
+                                user={props.user}
+                                onSave={props.onSave}
+                                onDelete={props.onDelete}
+                            />
+                        </div>
+                    )}
```

Comment:
What's the reason for this change?

Reply:
For props, most of the state handling stuff is handled by react-query so some props were no longer needed.

For the wrapping div, I'm honestly not sure why I put that there 😅. I just took it out and it made zero difference so I'll go ahead with removing it.